blockchain/blockchain.go

```diff
@@ -27,7 +27,7 @@ const (
 )
 
 var (
-	ErrNoBlock              = errors.New("no block data passed in")
+	ErrNoBlock              = errors.New("no block meta data passed in")
```

Comment:
This should be reverted:

```suggestion
	ErrNoBlock              = errors.New("block not found")
```